### ISSUE TITLE
Text with special character

### ISSUE THREAD

**Issue by Franckapik** [Shriinivas/blenderstrokefont#1]
Hi !

I come here to say first of all that this addon is a great idea and a good job!!

I wanted to know if it is possible to be able to import text with accents without having issues like this?

![text](https://user-images.githubusercontent.com/21478769/132316350-62815ab5-5fe1-46f7-89ec-3b6d8f4f6846.png)

I know this is maybe specific to certains languages as french for me ... but it could be a nice update ! 

Thank you !

F.

**Comment by Shriinivas** [comment]
Yes, it is possible. But currently the font you are using does not contain these letters with french accents. 
Maybe, if you have time you can add them yourself by editing the fonts. I have created a set of Inkscape extensions for editing the font. To add accented letters you would use the Edit Stroke Font extension, copy the letter without the accent and add the diacritical marks. For example: to create é, you will need to add accent aigu to e.

I have already done this for German letters (https://github.com/Shriinivas/etc/blob/master/inkscapestrokefont/fontsvgs/hershey-sans-umlaut.svg)

If interested you can go through these videos that explain the Inkscape stroke font extensions. https://www.youtube.com/playlist?list=PLxsh4i5F_h9E1pJDMRb4mfunzDpq2U2Ku

**Comment by joemarshall** [comment]
Hey, this is probably a bit late for you, but I have a PR which adds the [relief single line](https://github.com/isdat-type/Relief-SingleLine?tab=readme-ov-file) font, which has a bunch of accent characters in it. It's also nicely laid out, kerned properly, and just generally looks a lot better than most of the previous single line fonts.
